interface/patient_file/summary/pnotes_full.php

```diff
@@ -276,469 +276,462 @@ function restoreSession() {
 }
 </script>
 </head>
-<body class="body_top">
+<body>
 
-<div class="container-fluid" id="pnotes"> <!-- large outer DIV -->
+<div class="container-fluid mt-3" id="pnotes"> <!-- large outer DIV -->
```

Comment:
This feels like it should be a `container` instead of a `container-fluid`

Reply:
Updated the commit :)